Dockerfile.cr

```
@@ -1,5 +1,6 @@
 # Istioctl source images
 FROM eu.gcr.io/kyma-project/external/istio/istioctl:1.14.4 AS istio-1_14_4
+FROM eu.gcr.io/kyma-project/external/istio/istioctl:1.15.0 AS istio-1_15_0
 
 # Build image
 FROM golang:1.18.5-alpine3.16 AS build
```

```suggestion
FROM golang:1.19.1-alpine3.15 AS build
```

It seems that Istio uses some functionality that was introduced in Go v.1.19

- https://pkg.go.dev/sync/atomic@go1.19.1#Bool
- https://pkg.go.dev/sync/atomic@go1.18.6#Value

---

Good catch!